sim/dex-moves.ts

```diff
@@ -655,6 +655,17 @@ export class DexMoves {
 			if (move.gen > this.dex.gen) {
 				(move as any).isNonstandard = 'Future';
 			}
+			if (this.dex.parentMod) {
+				const parentMod = this.dex.mod(this.dex.parentMod);
```

Comment:
```suggestion
				// if move is exactly identical to parentMod's move, just use the old copy
				const parentMod = this.dex.mod(this.dex.parentMod);
```

Should probably have a comment so this is clear that's what's going on.